geode-core/src/main/java/org/apache/geode/management/internal/beans/MemberMBeanBridge.java

```diff
@@ -928,7 +928,7 @@ public String fetchLog(int numLines) {
         }
       } else {
         Assert.assertTrue(false,
-            "TailLogRequest/Response processed in application vm with shared logging.");
+            "TailLogRequest/Response processed in application vm with shared logging. This would occur if there is no 'log-file' defined.");
```

Comment:
actually, just wondering, instead of using a Assert statement that's bound to fail, should we just throw an IllegalStateException?

Reply:
Yes, good idea.